pkg/apis/pmemcsi/base/defaults.go

```diff
@@ -0,0 +1,85 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package base
+
+import (
+	corev1 "k8s.io/api/core/v1"
+)
+
+const (
+	// DefaultLogLevel default logging level used for the driver
+	DefaultLogLevel = uint16(5)
+	// DefaultImagePullPolicy default image pull policy for all the images used by the deployment
+	DefaultImagePullPolicy = corev1.PullIfNotPresent
+
+	defaultDriverImageName = "intel/pmem-csi-driver"
+	defaultDriverImageTag  = "canary"
+	// DefaultDriverImage default PMEM-CSI driver docker image
+	DefaultDriverImage = defaultDriverImageName + ":" + defaultDriverImageTag
+
+	// The sidecar versions must be kept in sync with the
+	// deploy/kustomize YAML files!
+
+	defaultProvisionerImageName = "k8s.gcr.io/sig-storage/csi-provisioner"
+	defaultProvisionerImageTag  = "v2.0.2"
+	// DefaultProvisionerImage default external provisioner image to use
+	DefaultProvisionerImage = defaultProvisionerImageName + ":" + defaultProvisionerImageTag
+
+	defaultRegistrarImageName = "k8s.gcr.io/sig-storage/csi-node-driver-registrar"
+	defaultRegistrarImageTag  = "v1.2.0"
+	// DefaultRegistrarImage default node driver registrar image to use
+	DefaultRegistrarImage = defaultRegistrarImageName + ":" + defaultRegistrarImageTag
+
+	// Below resource requests and limits are derived(with minor adjustments) from
+	// recommendations reported by VirtualPodAutoscaler(LowerBound -> Requests and UpperBound -> Limits)
+
+	// DefaultControllerResourceRequestCPU default CPU resource request used for controller driver container
+	DefaultControllerResourceRequestCPU = "12m" // MilliSeconds
```

Comment:
Please document under DEVELOPMENT.md how these defaults were obtained. I already added instructions for running Vertical Pod Autoscaler; what's missing is a description of the workload and how to run it.

You said that you used a script. In that case we'll need the script so that others can reproduce your experiment. It might be simpler to use the late binding stress test.

Reply:
Running the stress test is a good enough example, but anyway I also added the alternate to run the script.